packages/components/pie-toggle-switch/src/index.ts

```diff
@@ -35,28 +38,35 @@ export class PieToggleSwitch extends RtlMixin(LitElement) implements ToggleSwitc
 
     render () {
         const {
+            aria,
             isChecked,
             isDisabled,
         } = this;
 
+        const toggleSwitchId = 'toggle-switch-description';
+
         return html`
             <label
                 data-test-id="toggle-switch-component"
                 class="c-toggleSwitch"
                 ?isChecked="${isChecked}"
-                ?isDisabled=${isDisabled}>
+                ?isDisabled=${isDisabled}
+                aria-label="${aria?.label || nothing}">
                 <input
+                    data-test-id="toggle-switch-input"
                     role="switch"
                     type="checkbox"
                     class="c-toggleSwitch-input"
                     .checked="${isChecked}"
                     .disabled="${isDisabled}"
-                    @change="${this.onToggleChange}">
+                    @change="${this.onToggleChange}"
+                    aria-describedby="${aria?.describedBy ? toggleSwitchId : ''}">
```

Comment:
above we use `nothing`. here an empty string - should they be consistent? Is this adding an empty attribute if the prop is empty?

Reply:
ah I thought I'd updated that 😕. I'll fix it up